ansible/files/aeternity.fact

```diff
@@ -16,7 +16,9 @@ def read_node_file_contents(filename):
 
 try:
     with open(os.devnull, 'w') as devnull:
-        peer_key = subprocess.check_output(node_path + "/bin/aeternity peer_key", shell=True, stderr=devnull).decode('utf-8')
+        peer_key = subprocess.check_output(node_path + "/bin/aeternity
+                                           peer_key", shell=True,
```

Comment:
is this split of the command intentional, it's hard to scan ?

Reply:
I improved the style.

Reply:
thanks